[docs] Adds monitoring doc (#32281)
expo/expo: docs/pages/guides/using-logrocket.mdx

---
title: Using LogRocket
maxHeadingDepth: 4
hideTOC: true
description: A guide on installing and configuring LogRocket for session replays and error monitoring.
---

import { Terminal } from '~/ui/components/Snippet';

> [LogRocket](https://logrocket.com) records user sessions and identifies bugs as your users use your app. You can filter sessions by update IDs and also connect to your LogRocket account on the Expo dashboard to get quick acccess to your app's session data.

Reviewer: ```suggestion
[LogRocket](https://logrocket.com) records user sessions and identifies bugs as your users use your app. You can filter sessions by update IDs and also connect to your LogRocket account on the Expo dashboard to get quick access to your app's session data.
```

Typo.

## Install and configure LogRocket

You can install the LogRocket SDK with the following command:

<Terminal cmd={['$ npx expo install @logrocket/react-native expo-build-properties']} />

> Then, in your **app.json**, include the LogRocket config plugin:

Reviewer: ```suggestion
Then, in your [app config](/workflow/configuration/), include the LogRocket config plugin:
```

See: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-appjsonappconfigjsonappconfigjsappconfigts

```json app.json
{
  "plugins": [
    [
      "expo-build-properties",
      {
        "android": {
          "minSdkVersion": 25
        }
      }
    ],
    "@logrocket/react-native"
  ]
}
```

> Finally, initialize LogRocket in your app in a top-level file, like **tsx app/\_layout.tsx**:

Reviewer: ```suggestion
Finally, initialize LogRocket in your app in a top-level file, like **app/\_layout.tsx**:
```
